accounts/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
Consider adding test cases to this file. Use `TestCase` to create tests that verify the behavior of your application's features and ensure they work as expected.